lib/common/jit-reflector.ts

```diff
@@ -12,14 +12,10 @@ export class JitReflector implements CompileReflector {
     return this._reflectionCapabilities.annotations(typeOrFunc);
   }
 
-  componentModuleUrl(type: any, cmpMetadata: Component): string { /* tslint:disable-line prefer-function-over-method */
-    return `./${stringify(type)}`;
-  }
+  componentModuleUrl = (type: unknown) => `./${stringify(type)}`;
```

Comment:
as far as I remember `unknown` doesn't work on older TS versions. But it is awesome!

Reply:
Good catch!